dataInfo:
  camera:
    color:
      names: ['left', 'right', 'front', 'image_top']
      topics: ['/camera/color/left/image_raw', '/camera/color/right/image_raw', '/camera/color/front/image_raw', '/camera/color/image_top/image_raw']
    depth:
      names: ['left', 'right', 'front']
      topics: ['/camera/depth/left/image_raw', '/camera/depth/right/image_raw', '/camera/depth/front/image_raw']
    pointCloud:
      names: ['left', 'right', 'front']
      topics: ['/camera/pointCloud/left', '/camera/pointCloud/right', '/camera/pointCloud/front']
  arm:
    jointState:
      names: ['puppetLeft', 'puppetRight', 'masterLeft', 'masterRight']
      topics: ['/arm/jointState/puppetLeft', '/arm/jointState/puppetRight', '/arm/jointState/masterLeft', '/arm/jointState/masterRight']
    endPose:
      names: ['puppetLeft', 'puppetRight', 'masterLeft', 'masterRight']
      topics: ['/arm/endPose/puppetLeft', '/arm/endPose/puppetRight', '/arm/endPose/masterLeft', '/arm/endPose/masterRight']
      orients: [false, false, false, false]
  localization:
    pose:
      names: ['pika']
      topics: ['/localization/pose/pika']
  gripper:
    encoder:
      names: ['puppetLeft', 'puppetRight', 'masterLeft', 'masterRight']
      topics: ['/gripper/encoder/puppetLeft', '/gripper/encoder/puppetRight', '/gripper/encoder/masterLeft', '/gripper/encoder/masterRight']
  imu:
    9axis:
      names: ['pika']
      topics: ['/imu/9axis/pika']
  lidar:
    pointCloud:
      names: []
      topics: []
  robotBase:
    vel:
      names: []
      topics: []
  lift:
    motor:
      names: []
      topics: []
